python/sklearn-digits/app.py

```diff
@@ -97,8 +97,9 @@ def evaluate(self, file_path: str) -> Tuple[EvaluateResult, List[EvaluateResultD
                 label_predict.append(result.label)
 
             accuracy = accuracy_score(label_gold, label_predict)
-            p_r_f = precision_recall_fscore_support(label_gold, label_predict)
-            res = EvaluateResult(num, accuracy, p_r_f[0].tolist(), p_r_f[1].tolist(), p_r_f[2].tolist(), {})
+            uniq_labels = list(set(label_gold))
```

Comment:
Is this label's order correct?

Reply:
In this sample, we need to determine the labels by hand to fit the gold label.

Example
```
labels = ["0","1","2","3","4","5","6","7","8","9"]
```

Reply:
@keigohtr 
I explicitly separated label and label_index in https://github.com/rekcurd/rekcurd-example/pull/13/commits/6159e4175a7437a20f1e016570cf14f0d2bf5209 .
( label order is based on `self.labels` )
Please review it